tox.ini

```diff
@@ -1,22 +0,0 @@
-[tox]
```

Comment:
It's common to keep tox as a glue for various CI setups + local development. All you need to do is to separate running it as `tox --notest` + `tox` so that deps and tests steps are easily separated. Example: https://github.com/pypa/setuptools/blob/df9157c/.github/workflows/python-tests.yml#L120-L135.

It's common to set a `TOXENV=python` env var in the matrix in order to have multiple jobs in a matrix without copy-paste: https://github.com/pypa/setuptools/blob/df9157c/.github/workflows/python-tests.yml#L49-L51

Reply:
Okey, added that.